Tests/X509Tests/SignatureTests.swift

```diff
@@ -12,7 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-import XCTest
+@preconcurrency import XCTest
```

Comment:
I'm not convinced this is right. What errors did this silence?

Reply:
These:

`Static property 'secKeyRSA' is not concurrency-safe because non-'Sendable' type 'SecKey?' may have shared mutable state; this is an error in the Swift 6 language mode`

Reply:
Ah, we have a missing import. Instead of putting `@preconcurrency` here, please add this:

```
#if canImport(Darwin)
@preconcurrency import Security
#endif
```